.github/workflows/publish-ci.yml

```diff
@@ -80,35 +80,37 @@ jobs:
             py/server/build/wheel/
             py/embedded-server/build/wheel/
             py/client/build/wheel/
+            py/client-ticking/build/wheel/
             web/client-api/types/build/*.tgz
 
       - name: Publish deephaven-core to PyPi
         if: ${{ startsWith(github.ref, 'refs/heads/release/v') }}
         uses: pypa/gh-action-pypi-publish@release/v1
         with:
```

Comment:
You need to add the id-token to the job for Trusted Publishers to work:
```
+    permissions:
+      # IMPORTANT: this permission is mandatory for trusted publishing
+      id-token: write
```
Take a look at their example diff: https://blog.pypi.org/posts/2023-04-20-introducing-trusted-publishers/

Reply:
Thanks